packages/now-client/src/create-deployment.ts

```diff
@@ -114,10 +114,24 @@ export default function buildCreateDeployment(version: number) {
     let configPath: string | undefined;
     if (!nowConfig) {
       // If the user did not provide a config file, use the one in the root directory.
-      configPath = fileList
-        .map(f => relative(cwd, f))
-        .find(f => f === 'vercel.json' || f === 'now.json');
-      nowConfig = await parseNowJSON(configPath);
+      const relativePaths = fileList.map(f => relative(cwd, f));
+      const hasVercelConfig = relativePaths.includes('vercel.json');
+      const hasNowConfig = relativePaths.includes('now.json');
+
+      if (hasVercelConfig) {
+        if (hasNowConfig) {
+          throw new DeploymentError({
+            code: 'conflicting_config',
+            message:
+              'Both "vercel.json" and "now.json" exist. Please delete the "now.json" file',
```

Comment:
```suggestion
              'Cannot use both a `vercel.json` and `now.json` file. Please delete the `now.json` file.',
```

Let's make this consistent with the ignore error message.